build/three.js

```diff
@@ -1704,7 +1704,7 @@ Vector3.prototype = {
 		if ( typeof m === 'number' ) {
 
 			console.warn( 'THREE.Vector3: setFromMatrixColumn now expects ( matrix, index ).' );
-			var temp = m
+			var temp = m;
```

Comment:
changes to THREE.js should go into THREE.js, not here. The built version in `build` gets automatically created through an npm script.